app/campaigns/campaigns.service.js

```diff
@@ -0,0 +1,30 @@
+const CampaignDAO = require('./campaigns.db')
+
+const CampaignService = {
+  getCampaigns,
+  getActiveCampaigns,
+  getCompleteCampaigns
+}
+
+async function getCampaigns () {
+  const campaigns = await CampaignDAO.getCampaigns()
+  return {
+    campaigns: campaigns
+  }
+}
+
+async function getActiveCampaigns () {
+  const campaigns = await CampaignDAO.getActiveCampaigns()
+  return {
+    campaigns: campaigns
+  }
+}
+
+async function getCompleteCampaigns () {
```

Comment:
I'd suggest to add jsdocs to every function. You can also enforce that in eslint